packages/markdown-pdf/src/PdfTransformer.js

```diff
@@ -83,12 +82,88 @@ class PdfTransformer {
      * @param {Buffer} input - pdf buffer
      * @param {string} [format] result format, defaults to 'concerto'. Pass
      * 'json' to return the JSON data.
+     * @param {object} [options] - the PDF parsing options
+     * @param {number} [options.paragraphVerticalOffset] - the vertical offset used to detect pararaphs (defaults to 1)
+     * @param {boolean} [options.preservePages] - whether to preserve page breaks (defaults to true)
      * @returns {promise} a Promise to the CiceroMark DOM
      */
-    async toCiceroMark(input, format = 'concerto') {
-        const plainText = await parsePdf(input).then(extractPlainText);
-        return this.ciceroMarkTransformer.fromMarkdown(plainText, format);
+    async toCiceroMark(input, format = 'concerto', options = { paragraphVerticalOffset: 1, preservePages: true }) {
+        return new Promise( (resolve, reject) => {
+            const pdfParser = new PDFParser(null, false);
+            const errorCallback = (errData) => reject(`PDF parsing failed with error ${errData.parserError}`);
+            const conversionCallback = (pdfData) => {
 
+                const document = {
+                    $class : 'org.accordproject.commonmark.Document',
+                    xmlns : pdfData.formImage.Id.Name,
```

Comment:
This `xmlns` part of the model is rather confusing. Not sure I know what it means in general.

Reply:
Agreed - the naming is bad. The idea was that this would be some sort of canonical name/id for the document itself.